daemon/lokinet.cpp

```diff
@@ -651,6 +632,17 @@ main(int argc, char* argv[])
 #ifndef _WIN32
   return lokinet_main(argc, argv);
 #else
+  if (auto hntdll = GetModuleHandle("ntdll.dll"))
+  {
+    if (GetProcAddress(hntdll, "wine_get_version"))
+    {
+      static const char* text = "Don't run lokinet in wine, aborting startup";
+      static const char* title = "Lokinet Wine Error";
+      MessageBoxA(NULL, text, title, MB_ICONHAND);
+      abort();
```

Comment:
```suggestion
      std::abort();
```
(This is a bug from previous code this was copied from)